File: hack india/twostrings.java
import java.util.Scanner;
class twostrings
{
    static int l;
    static String twostring(String s1,String s2)
    {
        for(int i=0;i<s1.length();i++)
        {
            for(int j=0;j<s2.length();j++)
            {
                if(s1.substring(i,i+1)==s2.substring(j,j+1))
                {
                    l=l+1;
                }
            }
        }
        if(l>1)
        {
            return "Yes";
        }
        else{
            return "No";
        }

    }
    public static void main(String[] args) 
    {
        Scanner sc=new Scanner(System.in);
        Scanner sc1=new Scanner(System.in);
        System.out.println("enter the no of test casess"); 
        int n=sc.nextInt();
        for(int i=0;i<n;i++)
        {
            String s1=sc.nextLine();
            String s2=sc1.nextLine();
            System.out.println(twostring(s1,s2));
        }       
    }
}

File: hack india/twostrings.py

def twoStrings(s1, s2):
    l=0
    for i in range(0,len(s1)):
        for j in range(0,len(s2)):
            if(s1[i]==s2[j]):
                l=l+1
    if(l>=1):
        return "YES"
    else:
        return "NO"       
n=int(input())            
for i in range(1,n+1):
    s1=input()
    s2=input()
    print(twoStrings(s1,s2))
